src/app/lib/views/settings_container.js

```diff
@@ -452,6 +453,15 @@
                     App.vent.trigger('settings:show');
                     break;
                 case 'movies_quality':
+                case 'defaultOsWindowFrame':
+                    let packageJson = jsonFileEditor(`package.json`);
+
+                    packageJson.get('window').frame = value;
+
+                    packageJson.save();
+
```

Comment:
Can we remove this empty line and two above? (yes nitpicking obviously :p everything else looks good but havent built yet)